requirements.txt

```diff
@@ -1,29 +1,31 @@
 # Autogenerated by 'make bs'
 # If the format has changed, update MIN_PIPENV_VERSION in Makefile
 -i https://pypi.org/simple
-babel==2.16.0; python_version >= '3.8'
+babel==2.17.0; python_version >= '3.8'
```

Comment:
Do you know why any of this changed? Maybe will go away after a rebase.

Reply:
Were you able to figure out why these diffs are showing?

Reply:
Oh I had reverted the changes to this file and actually thought these changes were not re-introduced